backend/routes/watchlater.route.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import { getAllVideosinWL, addVideoToWL, removeVideo, clearWL } from '../controllers/watchlater.controller.js'
+const router = express.Router();
+
+router.get('/', getAllVideosinWL);
+router.post('/', addVideoToWL);
+router.delete('/:videoID', removeVideo);
+router.delete('/', clearWL);
```

Comment:
The naming convention of `removeVideo` is not consistent with the rest. You could use WatchLater instead of WL

Reply:
done :)